Classes/BEMCheckBox.h

```diff
@@ -82,6 +82,10 @@ typedef NS_ENUM(NSInteger, BEMAnimationType) {
  */
 @property (nonatomic) IBInspectable CGFloat lineWidth;
 
+/** The corner radius of the squared box. Default to 3.0.
+ */
+@property (nonatomic) IBInspectable CGFloat squaredBoxCornerRadius;
```

Comment:
I think that simply calling this property `conerRadius` would be less verbose.
We also need to clarify in the comments that `boxType` needs to be set to `BEMBoxTypeSquare` for this property to have any effect.